domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -672,6 +672,35 @@ class TopicListControllerTest {
     )
   }
 
+  @Test
+  fun testStoryList_markLessons_progressSaved_progressNotSaved_startedNotComp_ongoingListIsCrt() {
```

Comment:
Ditto here for ``Crt`` per my earlier comment. Consider:

``testStoryList_markLessons_progressSaved_notSaved_startedNotDone_ongoingListCorrect``

Would that work?

Reply:
yes that works, changed the test name